trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/InvariantResultTransformer.java

```diff
@@ -0,0 +1,50 @@
+package de.uni_freiburg.informatik.ultimate.pea2boogie;
```

Comment:
Please add a license header to each new file.

Reply:
I added a license to the file, does that look correct?